peer_mentoring/account_management/signals.py

```diff
@@ -0,0 +1,16 @@
+from .models import CustomUser, UserProfile
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+
+@receiver(post_save, sender=CustomUser)
```

Comment:
having this in models.py I think it's redundant to have it here twice

Reply:
I only had it twice so that I could get feedback on where it's appropriate. I have a profile project (that I'm almost done with) and I found recs on SO that recommended a signals.py file. If you think this is better in models then I'll delete.

Reply:
I am ok either way. signals.py might be a bit better organized indeed, but you do need to import it for it to auto-run right?